cristin-organization/src/test/java/no/unit/nva/cristin/organization/query/QueryCristinOrganizationHandlerTest.java

```diff
@@ -194,6 +199,40 @@ void shouldReturnVersionOneAsDefault() throws Exception {
         assertThat(gatewayResponse.getStatusCode(), equalTo(HTTP_OK));
     }
 
+    @Test
+    void shouldReturnCorrectPayloadWhenRequestingVersion20230526() throws Exception {
+        var fakeQueryResponseResource = IoUtils.stringFromResources(Path.of(CRISTIN_QUERY_RESPONSE_V_2_JSON));
+        doReturn(new HttpResponseFaker(fakeQueryResponseResource))
+            .when(cristinOrgApiClient20230526).fetchQueryResults(any());
+
+        queryCristinOrganizationHandler = new QueryCristinOrganizationHandler(clientProvider, new Environment());
+        var input =
+            generateValidHandlerRequestWithVersionParam(String.format(ACCEPT_HEADER_EXAMPLE, VERSION_2023_05_26));
+        queryCristinOrganizationHandler.handleRequest(input, output, context);
+
+        var gatewayResponse = GatewayResponse.fromOutputStream(output,
```

Comment:
This has nothing to with your code, just something that bothers me, with unneccessary code.
Wouldn't it be nice to skip the second Class ref, as you allready supplied it in the line before ?